client/lib/importer/test/api-payloads/defunct-importer.json

```diff
@@ -0,0 +1,6 @@
+{
+  "importId": "defunctImporter",
+  "type": "wordpress",
+  "importStatus": "importStopped",
+  "siteId": 0
+}
```

Comment:
I'm not sure if this was an accident or not. I usually try to add the newlines but sometimes don't because it seems like for the most part, node apps don't care. Added in 843e62c89f29830477508d8b80469cb53d3e39da